### ISSUE TITLE
Not seeing code compare 

### ISSUE THREAD

**Issue by jonprow** [lalit2054/yuyutsu#4]
**App Version: 1.0.2**
Platform: win32 10.0.19041
OS: Windows

![2021-10-14_7-06-10](https://user-images.githubusercontent.com/35116338/137306066-5a041182-1b37-4035-bed4-2caae426a8e2.png)
)

**Comment by SYBRWR** [comment]
Hi,
I'm facing the same issue. In my case also only the differences in the versions are shown.
I have cleared \.yuyutsu\artifacts and \.yuyutsu\temo without an other result.
i tried to compare iFlow from Neo platform.

**Comment by SYBRWR** [comment]
Yes I was able to transport the artifact but after the transport the compare with an other tenant shows only the version difference.
